static/css/cf-enhancements.less

```diff
@@ -761,6 +761,38 @@ tbody {
     }
 }
 
+/* topdoc
+  name: Steps list modifier
+  family: cfgov-cf-enhancements
+  notes:
+    - "Set up a list with 'Step x:' instead of bullets "
+  patterns:
+    - name: Default example
+      markup: |
+        <ol class="list list__steps short-desc">
+            <li class="list-item">Fill out the form</li>
+            <li class="list-item">Generate a PDF</li>
+            <li class="list-item">Email, fax or mail the PDF</li>
+        </ol>
+  tags:
+    - cf-typography
+*/
+
+.list__steps {
+    padding-left: 0;
+    counter-reset: item;
+    list-style: none;
+
+    .list-item {
+        position: relative;
+
+        &::before {
+            content: "Step " counter(item) ": ";
```

Comment:
Verify content is accessible on IE8. This "Step #" will likely not appear, but does that matter? Probably not.